src/Address.purs

```diff
@@ -47,20 +48,26 @@ addressToOgmiosAddress = addressBech32
 -- `Address` to `ScriptHash`
 --------------------------------------------------------------------------------
 -- | Get the `ScriptHash` with an internal `Address`
-addressScriptHash :: Address -> Maybe ScriptHash
-addressScriptHash = stakeCredentialToScriptHash <=< addressPaymentCred
```

Comment:
We lose a lot of generality in this module's API. I wonder if  using `enterpriseAddressToAddress >=> addressScriptHash` isn't a better option.